Formula/a/aicommit.rb

```diff
@@ -0,0 +1,20 @@
+class Aicommit < Formula
+  desc "AI-powered commit message generator"
+  homepage "https://github.com/coder/aicommit"
+  url "https://github.com/coder/aicommit/archive/refs/tags/v0.6.3.tar.gz"
+  sha256 "f42fac51fbe334f4d4057622b152eff168f4aa28d6da484af1cea966abd836a1"
+  license "CC0-1.0"
+
+  depends_on "make" => :build
+  depends_on "go" => "1.21"
```

Comment:
```suggestion
  depends_on "go"
```
Adding `=> "1.21"` doesn't do anything.